rioting-daisies/BangazonWorkforce
# Create Employee

This pull request is for the code that responsible for allowing the user to create a new employee. Once the Index view is created, the user can access this view by clicking Create New link.  For now, we can access this feature at http://localhost:5000/Employees/Create





Fixes Issue # 2

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. git fetch and checkout bj-employee
1.  run Bangazon and navigate to http://localhost:5000/Employees/Create
1. Once on the create page, the user should be able to create a new employee and the employee should be added to the database. 


# Checklist:
